src/components/Tag/Tag.tsx

```diff
@@ -10,40 +10,73 @@ import styles from './Tag.scss';
 
 export interface NonMutuallyExclusiveProps {
   /** Content to display in the tag */
-  children?: string;
+  children?: React.ReactNode;
   /** Disables the tag  */
   disabled?: boolean;
-  /** Callback when tag is clicked or keypressed. Renders without remove button when set. */
+  /** Callback when tag is clicked or keypressed. Renders without remove button or url when set. */
   onClick?(): void;
   /** Callback when remove button is clicked or keypressed. */
   onRemove?(): void;
+  /** A string to use when tag has more than textual content */
+  accessibilityLabel?: string;
+  /** Url to navigate to when tag is clicked or keypressed. */
+  url?: string;
 }
 
 export type TagProps = NonMutuallyExclusiveProps &
   (
-    | {onClick?(): void; onRemove?: undefined}
-    | {onClick?: undefined; onRemove?(): void}
+    | {onClick?(): void; onRemove?: undefined; url?: undefined}
+    | {onClick?: undefined; onRemove?(): void; url?: string}
   );
 
-export function Tag({children, disabled = false, onClick, onRemove}: TagProps) {
+export function Tag({
+  children,
+  disabled = false,
+  onClick,
+  onRemove,
+  accessibilityLabel = '',
+  url,
+}: TagProps) {
   const i18n = useI18n();
 
+  const segmented = onRemove && url;
```

Comment:
This seems like it could cause bugs in web. Can't we have segmented tags without urls?

Reply:
I think if you're using `<Stack>` you'll need to also do `wrap={false}` or if you're not, you would take care of making that custom content inline, if that's what you were looking to do from a user of `Tag`

Reply:
Ah I see what you mean. I think I just thought this line was doing more than it actually was. Once I 🎩 ed it in the playground it seems pretty much fine, and not a breaking change. I did notice some weirdness around styles not being added: [screencast](https://screenshot.click/16-59-ly7cm-pm84g.mp4)

<details>
<summary>Here's my playground</summary>
```js

import React from 'react';
import {Page, Stack, Tag} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      <Stack>
        <Tag onRemove={() => {}} url="#">
          tag with url
        </Tag>
        <Tag onRemove={() => {}}>tag with just onRemove</Tag>
      </Stack>
    </Page>
  );
}
```
</details>